packages/block-library/src/block/editor.scss

```diff
@@ -14,3 +14,45 @@
 		display: none;
 	}
 }
+
+// Styles for save button
+.components-toolbar-group {
+	.block-library-block__reusable-block-save-button {
+		margin: 7px;
+		height: 32px;
+		&::before {
+			left: 0 !important;
+			right: 0 !important;
+			height: 32px !important;
```

Comment:
There are some grid unit scss variables that should be used in the _variables.scss file (`$grid-unit-40`) for this one and line 22. (edit: I see now they're used elsewhere in the PR, so I guess this was just missed 👍 )

Would be good to add a comment above these lines about why `!important` is needed.